cmd/storage-rest-client.go

```diff
@@ -151,6 +152,28 @@ func (client *storageRESTClient) Hostname() string {
 	return client.endpoint.Host
 }
 
+func (client *storageRESTClient) UpdateBloomFilter(ctx context.Context, oldest, current uint64) (*bloomFilterResponse, error) {
+	b, err := json.Marshal(bloomFilterRequest{
```

Comment:
can we use gob instead of json as we use gob everywhere for inter-node communication?